elftools/dwarf/callframe.py

```diff
@@ -225,13 +428,16 @@ class CFIEntry(object):
         offset: the offset of this entry from the beginning of the section
         cie: for FDEs, a CIE pointer is required
```

Comment:
Can you document the augmentation_* fields here? With a link to the best online resource, if possible

Reply:
Ok. I can’t find much information about augmentation in the DWARF spec, so I’ll refer to Ian’s blog.